RIAssigner/data/SimpleData.py

```diff
@@ -2,18 +2,20 @@
 from .Data import Data
 import numpy as np
 
-class NumpyData(Data):
+from copy import copy
+
+class SimpleData(Data):
     """Class to handle data from numpy arrays
     """
 
-    def __init__(self, rt: np.ndarray):
+    def __init__(self, retention_times: Iterable[float]):
         """Constructor for `NumpyData` class.
 
         Args:
             rt (np.array): Retention time values
         """
         super().__init__(None, None, "sec")
-        self._retention_times = rt.copy()
+        self._retention_times = copy(retention_times)

```

Comment:
Is there any reason to prefer built-in `copy()` over that of NumPy? If anything I'd assume that `ndarray.copy()` would perform faster.